I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 37, in script
    play ambience "audio/Feather Waltz.mp3" fadein 5.0 loop volume 2.0
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
Exception: Audio channel 'ambience' is unknown.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 37, in script
    play ambience "audio/Feather Waltz.mp3" fadein 5.0 loop volume 2.0
  File "C:\Users\gunin\Documents\renpy-8.1.3-sdk\renpy\ast.py", line 2259, in execute
    self.call("execute")
  File "C:\Users\gunin\Documents\renpy-8.1.3-sdk\renpy\ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\gunin\Documents\renpy-8.1.3-sdk\renpy\statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "C:\Users\gunin\Documents\renpy-8.1.3-sdk\renpy\audio\music.py", line 103, in play
    c = get_channel(channel)
  File "C:\Users\gunin\Documents\renpy-8.1.3-sdk\renpy\audio\audio.py", line 947, in get_channel
    raise Exception("Audio channel %r is unknown." % name)
Exception: Audio channel 'ambience' is unknown.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
starwrecked 1.0
Sun Jan 14 15:55:16 2024
